src/components/FriendCard/friendcards.js

```diff
@@ -1,55 +1,84 @@
 import React from "react";
+import Parse from "parse/dist/parse.min.js";
 import "./friendcards.css";
 
-function FriendCard() {
-	const titles = ["Bob Dylan", "Cate Blanchett"];
-	const texts = ["1 friend", "2 friends"];
-
-	return (
-		<div>
-			{titles.map((title, index) => (
-				<Card
-					key={index}
-					title={title}
-					text={texts[index]}
-				/>
-			))}
-		</div>
-	);
+function FriendCards({
+  filteredUsers,
+  friendNames,
+  addedNames,
+  currentUserName,
+}) {
+  return (
+    <div>
+      {filteredUsers.map((user, index) => (
+        <Card
+          key={index}
+          user={user}
+          currentUserName={currentUserName}
+          isFriend={(friendNames ?? []).includes(user)}
+          isAdded={(addedNames ?? []).includes(user)}
+        />
+      ))}
+    </div>
+  );
 }
 
-function Card({ title, text }) {
-	const handleRemove = () => {
-		// Implement the logic for removing the friend
-		console.log(`Removing friend: ${title}`);
-	};
-
-	return (
-		<div className="friend-card">
-			<div className="friend-content">
-				<div className="image-container">
-					<div className="profile-image-border">
-						<div className="profile-image-wrapper">
-							<img
-								src={"/images/bob.jpeg"}
-								alt="Profile"
-								className="profile-image"
-							/>
-						</div>
-					</div>
-				</div>
-				<div className="text-container">
-					<h1 className="friend-name">{title}</h1>
-					<h1 className="friend-text">{text}</h1>
-				</div>
-				<button
-					className="remove-button"
-					onClick={handleRemove}>
-					Remove
-				</button>
-			</div>
-		</div>
-	);
+function Card({ user, isFriend, currentUserName, isAdded }) {
+  const handleAction = async () => {
+    if (isFriend) {
```

Comment:
@pantageepapa because when you are already friends, the button will display "Remove" (that's the only action you can take). If you're not friends (isFriends is false) the action will be add! Realised I need to make it an if, elif, else statement though cause the "Added" button shouldn't do anything, but I'll change that for sprint 3 :)

Reply:
Ah okay makes sense!